### ISSUE TITLE
balance() does not move the engines

### ISSUE THREAD

**Issue by ngomezve** [MIT-LAE/TASOPT.jl#11]
balance() changes the wing box location (xwbox) to trim the aircraft but the engines remain in the initial user-defined position.

**Comment by askprash** [comment]
Yes that's right. It's something that's been on my list for a bit. As it is currently implemented the engine locations are set in absolute coordinates  and it is self consistent (with the assumption being that the pylons grow/shrink to accommodate the `xwbox` movement. This is fine if the movement is ~10cm but otherwise perhaps not realistic since we don't model the pylon weight increase/ decrease etc ). 

A possible solution is to define the engine locations relative to the wing Xbox. This requires some change to the current arrays and not sure if worth a hot fix given plans to structure-ify the different components.
